gdsc-web-backend/Controllers/PagesController.cs

```diff
@@ -9,34 +10,37 @@ namespace gdsc_web_backend.Controllers
     [Route("api/[controller]")]
     public class PagesController : ControllerBase
     {
-        public List<PageModel> pageModels = new List<PageModel>
+        private readonly List<PageModel> _mockFaq = new();
```

Comment:
This variable should be named `_mockPages`, not `_mockFaq`. Use `Shift + F6` to name all code occurencies